typhos/tests/test_cli.py

```diff
@@ -38,6 +39,9 @@ def test_cli_no_entry(monkeypatch, qtbot, happi_cfg):
 
 def test_cli_stylesheet(monkeypatch, qapp, qtbot, happi_cfg):
     monkeypatch.setattr(QApplication, 'exec_', lambda x: 1)
+    monkeypatch.setattr(
+        pydm.exception, 'raise_to_operator', lambda *_, **__: None
+    )
```

Comment:
This makes a lot of sense. In a future edit it might make sense to direct raise_to_operator directly to raise inside the test suite.